.env.rc0

```diff
@@ -0,0 +1,150 @@
+# Don't use "//" for comments in this file.
+# This file is meant to be executed as a bash script for testing.
+ENV_TYPE="production"
+
+GETH_VERBOSITY=2
+
+VM_BASED=true
+
+KUBERNETES_CLUSTER_NAME="releasecandidate0"
+KUBERNETES_CLUSTER_ZONE="us-west1-a"
+CLUSTER_DOMAIN_NAME="celo-testnet"
+
+TESTNET_PROJECT_NAME="celo-testnet-production"
+
+BLOCKSCOUT_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/blockscout"
+BLOCKSCOUT_DOCKER_IMAGE_TAG="982ffa2a1ddba3b5ccfe0915bd902ab8671ddc99"
+BLOCKSCOUT_WEB_REPLICAS=3
+# Increment this value everytime you redeploy blockscout. Or else the deployment will fail due to the
+# existing database.
+BLOCKSCOUT_DB_SUFFIX="1"
+BLOCKSCOUT_SUBNETWORK_NAME="RC0"
+
+CELOSTATS_SERVER_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/celostats-server"
+CELOSTATS_SERVER_DOCKER_IMAGE_TAG="2d970c69c109ab4089bddfe50c8f75d872e6e3a3"
+CELOSTATS_FRONTEND_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/celostats-frontend"
+CELOSTATS_FRONTEND_DOCKER_IMAGE_TAG="db16a1a4d9882cfed93ffa9f61cfa9f80cadfe18"
+CELOSTATS_TRUSTED_ADDRESSES=""
+CELOSTATS_BANNED_ADDRESSES=""
+CELOSTATS_RESERVED_ADDRESSES=""
+
+GETH_NODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/geth"
+# When upgrading change this to latest commit hash from the master of the geth repo
+# `geth $ git show | head -n 1`
+GETH_NODE_DOCKER_IMAGE_TAG="2d730430a7e22f5890dfb6ac74cb9f7297a3630d"
+
+GETH_BOOTNODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/geth-all"
+# When upgrading change this to latest commit hash from the master of the geth repo
+# `geth $ git show | head -n 1`
+GETH_BOOTNODE_DOCKER_IMAGE_TAG="2d730430a7e22f5890dfb6ac74cb9f7297a3630d"
+
+CELOTOOL_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/celo-monorepo"
+CELOTOOL_DOCKER_IMAGE_TAG="celotool-9548656d0411d8c3b3d525727e621442e21b055a"
+
+CELOCLI_STANDALONE_IMAGE_REPOSITORY="gcr.io/celo-testnet/celocli-standalone"
+CELOCLI_STANDALONE_IMAGE_TAG="0.0.30-beta2"
+
+# Schedule for an oracle deployed via celotool, expressed in crontab syntax
+# This schedule is "every 5th minute"
+ORACLE_CRON_SCHEDULE="*/5 * * * *"
```

Comment:
We won't have an oracle for RC0, is there an option to deploy without oracles? cc @yerdua @tkporter

Reply:
Quick search yields some kinda hacky ways to basically set a cron job that will never execute (i.e. set it to run on feb 31). Let me know if I should do this for oracle and voting_bot
https://stackoverflow.com/questions/8324306/cron-job-that-will-never-execute

Reply:
The oracle is its own helm chart that is deployed using a different command, so we can just not deploy it and be totally fine. So we can remove this line or keep it, doesn't matter